packages/ui/src/components/layout/LayoutContent.tsx

```diff
@@ -0,0 +1,14 @@
+import type { ReactNode } from 'react';
+
+import { cn } from '@/utils';
+
+import { layoutDef } from './Layout.styles';
+
+interface LayoutContentProps {
+  children: ReactNode;
+  className?: string;
+}
+
+export function LayoutContent({ children, className }: LayoutContentProps) {
```

Comment:
můžeme spíš všude použít `ComponentPropsWithoutRef<'div'>` a zároveň spreadnout `...restProps` nad divem, tak ať kdyby náhodou tam můžeš poslat jakékoliv propsy a tady se to nemusí dplňovat?

Reply:
koukám, že už si to změnil